src/Hypernode/Magento/Command/Hypernode/Modules/ListUpdatesCommand.php

```diff
@@ -65,42 +68,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
-        $modules = $this->getModules()
-                ->filterModules($input);
+        $onlyActive = $input->getOption('only-active') ? true : false;
```

Comment:
I think you can do `boolval($input->getOption('only-active'))` to achieve the same